product_template_websale_search_extended/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+{
+    "name": "Product Template Websale Search Extended",
+    "summary": """
+        Adds new feature to make searching by different fields easier.
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "12.0.1.0.0",
+    "category": "Extra Tools",
```

Comment:
Better category IMO:
```suggestion
    "category": "Website",
```